pluma/core/baseclasses/consolebase.py

```diff
@@ -58,17 +58,25 @@ def read_all(self, preserve_read_buffer: bool = False) -> str:
         self.require_open()
         return self.engine.read_all(preserve_read_buffer=preserve_read_buffer)
 
-    def wait_for_match(self, match: List[str], timeout=None) -> str:
+    def wait_for_match(self, match: List[str], timeout=None) -> Optional[str]:
         '''Wait a maximum duration of 'timeout' for a matching regex, and returns matched text'''
         self.require_open()
         match_result = self.engine.wait_for_match(match=match, timeout=timeout)
         return match_result.text_matched
 
-    def wait_for_bytes(self, timeout: int = None, sleep_time: int = None,
+    def wait_for_bytes(self, timeout: Union[int, float] = None,
+                       sleep_time: Union[int, float] = None,
                        start_bytes: int = None) -> bool:
```

Comment:
The `int` type is included in the `float` type:
```suggestion
    def wait_for_bytes(self, timeout: Optional[float] = None,
                       sleep_time: Optional[float] = None,
                       start_bytes: Optional[int] = None) -> bool:
```

Reply:
Haha of course...😌 thanks